dom-event-delegation/main.js

```diff
@@ -0,0 +1,12 @@
+var list = document.querySelector('.task-list');
+list.addEventListener('click', handleClick);
+function handleClick(event) {
+  console.log('event.target:', event.target);
+  console.log('value of event.target.tagName:', event.target.tagName);
+  if (event.target.tagName === 'BUTTON') {
```

Comment:
this is the reverse of what you want, clicking on the button should remove the task item, and only clicking on the button.